Packages/com.unity.inputsystem/InputSystem/Actions/Composites/Vector2Composite.cs

```diff
@@ -124,7 +124,7 @@ public class Vector2Composite : InputBindingComposite<Vector2>
         /// </code>
         /// </example>
         /// </remarks>
-        public Mode mode;
+        public Mode mode = Mode.DigitalNormalized;
```

Comment:
Is this a API/behaviorial change (may not be a valid patch) or is it just a case of explicit initialisation? (fine)

Reply:
I answer my own question, it seems like its not a change since Mode.DigitalNormalized was assigned value zero which would be equivalent to default(Mode), hence this is better in my opinion since explicit.

Reply:
I must admit, I'm a little sad that `Vector2Composite` and `Vector3Composite` both have a `Mode` enum with the same values (although they have been re-ordered for Vector2) and that the mode defaults are different.

But yes, this is just a cosmetic change for explicitness and consistency.

Reply:
Yes that inconsistency is indeed sad, something to address when a breaking change can be done.